src/components/forms/controlled-form/controlled-password-input/controlled-password-input.tsx

```diff
@@ -0,0 +1,78 @@
+import type { FC, InputHTMLAttributes } from "react";
+import { useState } from "react";
+import { type UseFormRegister, type UseFormWatch } from "react-hook-form";
+
+import { PasswordStrengthIndicator } from "@/components/forms/controlled-form/controlled-password-input/password-strength-indicator";
+import { Button } from "@/components/ui/button";
+import { Input } from "@/components/ui/input";
+import { Label } from "@/components/ui/label";
```

Comment:
Consider using a barrel file in ui to simplify imports.
```suggestion
import { Button, Input, Label } from "@/components/ui";
```